packages/cli/src/commands/sandbox/sandbox-secret/sandbox_secret_list_command.test.ts

```diff
@@ -98,10 +99,11 @@ void describe('sandbox secret list command', () => {
     });
 
     assert.equal(printMock.mock.callCount(), 1);
-    assert.ok(
-      printMock.mock.calls[0].arguments[0].startsWith(
-        'No sandbox secrets found.'
-      )
+    assert.equal(
+      printMock.mock.calls[0].arguments[0],
+      `No sandbox secrets found. To create a secret use ${cyan(
```

Comment:
you can use `format.command` here instead of depending on `kleur` here

Reply:
even better, use `format.normalizeAmpxCommand` to automatically format and prefix with the package manager currently in use

Reply:
Thank you for your suggestion.
I use `format.normalizeAmpxCommand` instead of  of depending on `kleur` here.